README.rst

```diff
@@ -1,140 +1,47 @@
 Google API Artifact Manager
 ===========================
 
-Google API Artifact manager (artman) is a set of modules used to automate the
-creation of software artifacts related to APIS defined using `protobuf`_ IDL.
+Google API Artifact Manager (``artman``) is a program used to automate the
+creation of API client libraries.
```

Comment:
generation and publishing (to align with `generate` and `publish` commands we have?)